### PR TITLE
Add to ethereum contract integration test coverage

### PR DESCRIPTION
**Pull request by claudijd**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/1053)

### CODE REVIEW

File: ethereum/README.md [hunk 1]
```diff
@@ -28,7 +28,11 @@ For each test run:
 
 Run `npx ganache-cli --deterministic --time "1970-01-01T00:00:00+00:00"` to start a chain.
 
-Run the tests using `npm run test`
+Run the all ethereum tests using `npx truffle test`
```
Comment (kcsongor) [inline ethereum/README.md:31]: Can we include the
```
make test
```
route as an alternative, which takes care of spinning up ganache and running the tests

File: ethereum/test/wormhole.js [hunk 2]
```diff
@@ -10,6 +10,13 @@ const testSigner1PK = "cfb12303a19cde580bb4dd771639b0d26bc68353645571a8cff516ab2
 const testSigner2PK = "892330666a850761e7370376430bb8c2aa1494072d3bfeaed0c4fa3d5a9135fe";
 const testSigner3PK = "87b45997ea577b93073568f06fc4838cffc1d01f90fc4d57f936957f3c4d99fb";
 
+const core = "0x00000000000000000000000000000000000000000000000000000000436f7265"
```
Comment (kcsongor) [inline ethereum/test/wormhole.js:13]: Might be more readable if
a) we put a comment here saying it's "Core" in ascii
b) construct the string programmatically, like
```
const core = '0x' + Buffer.from("Core").toString("hex").padStart(64,0)
```

Comment (claudijd) [reply ethereum/test/wormhole.js:13]: I like option B, I'll do that.

File: ethereum/README.md (hunk 1 above)
Comment (claudijd) [reply ethereum/README.md:31]: yep, 💯

File: ethereum/test/wormhole.js [hunk 3]
```diff
@@ -483,10 +515,16 @@ contract("Wormhole", function () {
         const initialized = new web3.eth.Contract(ImplementationFullABI, Wormhole.address);
         const accounts = await web3.eth.getAccounts();
 
-        let data = "0x00000000000000000000000000000000000000000000000000000000436f726504";
-        data += [
+        data = [
+            // Core
+            core,
+            // Action 4 (set fees)
+            "04",
```
Comment (kcsongor) [inline ethereum/test/wormhole.js:522]: `actionTransferFee`?

**Review by bruce-riley — APPROVED** [review]

**Review by bruce-riley — APPROVED** [review]

**Review by bruce-riley — APPROVED** [review]

**Review by bruce-riley — APPROVED** [review]

**Review by evan-gray — APPROVED** [review]

**Review by bruce-riley — APPROVED** [review]

**Pull request merged**